rerun_py/tests/unit/test_dataframe.py

```diff
@@ -459,7 +459,12 @@ def any_value_static_recording(tmp_path: pathlib.Path) -> rr.dataframe.Recording
 
     with rr.RecordingStream(APP_ID, recording_id=uuid.uuid4()) as rec:
         rec.save(rrd_path)
-        rec.log("/test", rr.AnyValues(yak="yuk", foo="bar", baz=42), static=True)
+        rec.log(
+            "/test",
+            # Note: defensive parameter names to avoid ANY_VALUE_TYPE_REGISTRY collisions
```

Comment:
This comment will become very hard to understand when we change the name of `ANY_VALUE_TYPE_REGISTRY`.
Better to explain "We use long, explicit names here because `AnyValue` will remember and check that future calls use the same type for the same name" or similar